### ISSUE TITLE
Auth Access Clobbering issue

### ISSUE THREAD

**Issue by dan-acsc** [CybercentreCanada/assemblyline#70]
**Problem**
When configuring auth as OAuth if you set access more than once, (e.g for a group and for an email) only the last one applied will actually work.

The other will have no effect because it will be overriden. The reason for this is the way the "access" check is implemented.

The following code found in assemblyline-ui/assemblyline_ui/helper/oauth.py
```python
        for auto_prop in provider.auto_properties:
            if auto_prop.type == "access":
                # Set default access value for access pattern
                access = auto_prop.value.lower() != "true"
```
This code effectively make it so each time a new access prop is seen all previous access props overriden by setting access to False if the user doesn't pass whichever check runs last.

**Solution**
Either change the way Access works, pre-check if there are any access props and set access to False if there are, then when applying access permissions just give access if any access check passes.

Alternatively document the current behaviour in the Auth section of the Installation documentation https://cybercentrecanada.github.io/assemblyline4_docs/installation/configuration/authentication/

**Comment by cccs-sgaron** [comment]
Fixed in this PR: https://github.com/CybercentreCanada/assemblyline-ui/pull/623

**Comment by cccs-rs** [comment]
Fixed in [Release 4.4.0.21](https://github.com/CybercentreCanada/assemblyline/releases/tag/v4.4.0.stable21)
